ONSdigital/florence
# Dataset/datset version review workflow

### What

Add the UI options to move a dataset/version through the review and approve workflow.

### How to review

Checkout Zebedee to the branch `feature/datasets-review-workflow`.

On both a dataset and a version/instance, you should be able to:
1) Go to a new version or a dataset and add `?collection=*collectionID*` to the end of the URL (in the future this should be passed through from the datasets screen).
2) 'Save and submit for review' should redirect to that collection on the collections screen and the item will have moved to the 'awaiting review' heading.
3) Go back to the dataset/version you just added, you should still only see options to save or save and submit for review, not 'submit for approval'.
4) Log in as a different user and go to that dataset/version, you should see a button to 'Save and submit for approval'.
5) You should be redirected to the collection again and the dataset/version should be in the 'Reviewed' heading.

🎉 

### Who can review

Probably Jonbo (@jondewijones)
